bot/pagination.py

```diff
@@ -55,15 +56,13 @@ def __init__(self, prefix='```', suffix='```',
         self._pages = []
 
     def add_line(self, line='', *, empty=False):
-        """Adds a line to the current page.
+        """
+        Adds a line to the current page.
 
-        If the line exceeds the :attr:`max_size` then an exception
-        is raised.
+        If the line exceeds the :attr:`max_size` then an exception is raised.
```

Comment:
```suggestion
        If the line exceeds the max_size then an exception is raised.
```

Reply:
Unsure about the `:attr:` declaration, but inline code and attributes I prefer to keep wrapped in backticks for correct inline code formatting, both for IDE output and for the possible future of having autogen docs.